components/post_view/floating_timestamp/floating_timestamp.jsx

```diff
@@ -46,7 +46,10 @@ export default class FloatingTimestamp extends React.PureComponent {
         }
 
         return (
-            <div className={className}>
+            <div
+                className={className}
+                data-testid='post-list__timestamp'
```

Comment:
I suggest to change into
```suggestion
                data-testid='floatingTimestamp'
```